world_gen.py

```diff
@@ -35,7 +35,11 @@ def getLevelRow():
         
         if roll == 4 and y < height - 1 - drunk['padding']:
             drunk['y'] += 1
-    
+
+    for y in range(width):
+        for x in range(height):
+            if random.randint(1,20) == 1 and level[x][y] == '1':
+                level[x][y] = 'E'
```

Comment:
This should probably happen in a different function, but we can leave it for now.